scripts/combobox.js.php

```diff
@@ -1,136 +0,0 @@
-<?php
```

Comment:
I removed a few CSS based on the removed code, which I don't find anywhere else in the plugin.